script/downgrade_bundler_on_old_rails

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# This file was generated on 2019-01-03T20:34:23+00:00 from the rspec-dev repo.
+# DO NOT modify it by hand as your changes will get lost the next time it is generated.
+
+set -e
+source script/functions.sh
+
+if is_ruby_23_plus; then
+  if ruby -e "exit(!(ENV['RAILS_VERSION'].to_f < 5))"; then
```

Comment:
I think there is a mistake. We don't need the bang.

With:
```sh
set -e

ruby -v
echo $RAILS_VERSION

function is_ruby_23_plus {
  if ruby -e "exit(RUBY_VERSION.to_f >= 2.3)"; then
    return 0
  else
    return 1
  fi
}

if is_ruby_23_plus; then
  if ruby -e "exit(ENV['RAILS_VERSION'].to_f < 5)"; then
    # On Rails versions less than 5, Bundler 2.0 is not supported
    echo "Warning dowgrading to older version of Bundler"
    gem uninstall bundler
    gem install bundler -v '1.17.3'
  else
    echo "Rails >= 5"
  fi
fi
```
output
```
ruby 2.5.3p105 (2018-10-18 revision 65156) [x86_64-darwin18]
~> 4.2.0
Warning dowgrading to older version of Bundler
```

Reply:
Yes a typo from a half way though